### ISSUE TITLE
NameError: name 'numpy' is not defined

### ISSUE THREAD

**Issue by brandtbucher** [brandtbucher/SpeakEasy#1]
```python
NameError: name 'numpy' is not defined
```

**Comment by brandtbucher** [comment]
Error is raised when attempting to make corrections.

**Comment by brandtbucher** [comment]
This is caused by lines `220` and `221`...

```python
            numpy.save("phrases", phrases)
            numpy.save("context", context)
```

...which were missed during a pre-release change from `NumPy` to `pickle` for object persistence.

**Comment by brandtbucher** [comment]
Fixed in `1.0.1`.

`220` and `221` have been replaced with:

```python
            with open("phrases.pickle", "wb") as file:
                pickle.dump(phrases, file, pickle.HIGHEST_PROTOCOL)
         
            with open("context.pickle", "wb") as file:
                pickle.dump(context, file, pickle.HIGHEST_PROTOCOL)
```
